### PR TITLE
Added RustScan to repo

### PR DESCRIPTION
**Pull request by brandonskerritt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ If you want to contribute, please read [this](CONTRIBUTING.md).
     - [Observability](#observability)
     - [Operating systems](#operating-systems)
     - [Productivity](#productivity)
-    - [Security tools](#security-tools)
```
Comment (palfrey) [inline README.md:20]: Why was this deleted?

File: README.md [hunk 2]
```diff
@@ -282,7 +281,8 @@ See also [A comparison of operating systems written in Rust](https://github.com/
...
 * [ripasso](https://github.com/cortex/ripasso/) — A password manager, filesystem compatible with pass
 * [kpcyrd/rshijack](https://github.com/kpcyrd/rshijack) — A TCP connection hijacker, rust rewrite of shijack [<img src="https://api.travis-ci.org/kpcyrd/rshijack.svg?branch=master">](https://travis-ci.org/kpcyrd/rshijack)
-* [kpcyrd/sniffglue](https://github.com/kpcyrd/sniffglue) — A secure multithreaded packet sniffer [<img src="https://api.travis-ci.org/kpcyrd/sniffglue.svg?branch=master">](https://travis-ci.org/kpcyrd/sniffglue)
+* [rustscan/rustscan](https://github.com/RustScan/RustScan) — Make Nmap faster with this port scanning tool [<img alt="Actions" src="https://github.com/RustScan/RustScan/workflows/Continuous%20integration/badge.svg?branch=master">](https://github.com/RustScan/RustScan/actions?query=workflow%3A%22Continuous+integration%22)
+* [kpcyrd/sniffglue](https://github.com/kpcyrd/sniffglue) — A secure multithreaded packet sniffer [<img alt="Actions" src="https://github.com/RustScan/RustScan/workflows/Continuous%20integration/badge.svg?branch=master">](https://travis-ci.org/kpcyrd/sniffglue)
```
Comment (palfrey) [inline README.md:285]: You've changed the build image for sniffglue. Can you please change it back?

File: README.md (hunk 1 above)
Comment (brandonskerritt) [reply README.md:20]: My bad! I made these changes in the GitHub web interface and the find command they used must have dome something. I'll fix these :)

**Comment by palfrey** [comment]
> Sorry about that, how about this? :)

Much better, thanks!
